### PR TITLE
Adding environment variable documentation

### PR DESCRIPTION
**Pull request by Bento007**

### CODE REVIEW

File: docs/environment/README.md [hunk 1]
```diff
@@ -33,15 +33,15 @@ Environment Variable | Description
...
 `API_DOMAIN_NAME` | The domain of the API host.
-`TOKENINFO_FUNC` | 
-`OIDC_AUDIENCE` | 
+`TOKENINFO_FUNC` | Do Not Modify. Used by connexion to verify the JWT in the authorization header of an authenticated request.
+`OIDC_AUDIENCE` | A list of allowed audiences in the JWT.
 `OPENID_PROVIDER` | 
-`OIDC_EMAIL_CLAIM` | 
-`OIDC_GROUP_CLAIM` | 
+`OIDC_EMAIL_CLAIM` | The OIDC claim that specifies the users email.
+`OIDC_GROUP_CLAIM` | The OIDC claim that specifies the Groups the users belongs.
```
[the remark this answers is not in the document]
Comment (Bento007) [reply docs/environment/README.md]: will fix

File: docs/environment/README.md [hunk 2]
```diff
@@ -33,15 +33,15 @@ Environment Variable | Description
...
 `NOTIFY_URL` | 
-`DSS_AUTHORIZED_GOOGLE_PROJECT_DOMAIN_ARRAY` | 
-`DSS_AUTHORIZED_DOMAINS` | 
-`DSS_AUTHORIZED_DOMAINS_TEST` | 
+`DSS_AUTHORIZED_GOOGLE_PROJECT_DOMAIN_ARRAY` | A list domains with authorization to perform restricted actions on the DSS. This is generally used for Google service account credentials.
```
[the remark this answers is not in the document]
Comment (Bento007) [reply docs/environment/README.md]: only by `DSS_AUTHORIZED_DOMAINS` to build the list of authorized domains

[the remark this answers is not in the document]
Comment (xbrianh) [reply docs/environment/README.md]: Where does this happen?

[the remark this answers is not in the document]
Comment (Bento007) [reply docs/environment/README.md]: https://github.com/HumanCellAtlas/data-store/blob/e3c5839ea335cb6b1bb14db0e73a818e96eff43c/environment#L84

**Pull request merged**